docs/core/entity/assist-satellite.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Assist satellites"
+---
+
+An Assist Satellite entity represents the Assist pipeline-powered voice assistant capabilities of a device. Devices with such entities can allow users to control Home Assistant using their voice.
+
+## Base entity
+
+The `AssistSatelliteEntity` base class must be inherited when implementing the `assist_satellite` platform. For example:
+
+```python
```

Comment:
We should use the following headers and fill in content for states and supported features. See the calendar integration page for example.

```suggestion
## Properties

### States

## Supported features

## Methods

```python
```